app/presenters/document_presenter.rb

```diff
@@ -0,0 +1,45 @@
+class DocumentPresenter < SimpleDelegator
+  delegate :manual, to: :document
+  attr_reader :document
+
+  def initialize(document:)
+    @document = document
+    super(document)
+  end
+
+  # TODO: Manual sections should be tagged to taxons like manuals are.
+  # Unfortunately, that workflow doesn't yet exist, which means we need to
+  # manually generate taxonomy breadcrumbs for manual sections by using the
+  # manual's' taxonomy breadcrumbs and appending the manual section metadata to
+  # the end.
+  # This code should be deleted once manual sections are tagged to the same
+  # taxons as the parent manual.
+  def taxonomy_breadcrumbs
+    @taxonomy_breadcrumbs ||= begin
+      document_breadcrumbs = nav_helper.taxon_breadcrumbs[:breadcrumbs]
+
+      # Remove the last element, which is the manual as `current`.
+      document_breadcrumbs.pop
+
+      # Re-add the manual entry, now as a regular breadcrumb
+      document_breadcrumbs << {
+        title: manual.title,
+        url: manual.url
+      }
+
+      # Add the section breadcrumb a `current`
+      document_breadcrumbs << {
```

Comment:
For "our" manuals (ones that go through manuals publisher) this works, as they can only be manual -> section, but for the hmrc manuals (ones that come from their publisher via hmrc-manauls-api) this won't work as it looks like it adds only the root (manual) and leaf (current document) nodes, but they can be arbitrarily nested within other documents (e.g. https://www.gov.uk/hmrc-internal-manuals/international-manual/intm162540 which has two intermediate parent documents between it and the manual).

I suspect this won't matter for the a/b test you are running as it's just education content, but it's probably safer to make sure we've got the right structure from the get go just in case.